### PR TITLE
Improves contribution guidance

### LINKED ISSUE #202: Improving contribution guidance on the project
It seems clear to me that the contribution guide is not perfect and is not providing enough guidance to new contributors. But it requires more than just this file to be able to contribute efficiently on the project. 

# Requirements

 - [x] Details about probes and scoring roles, locations and how they are ran.
 - [x] GitHub issue templates for
     - [x] new probe, what is it we want to retrieve and how it should be used
     - [x] bug report, what is the behavior observed and what was expected
 - [x] Pull request template listing
     - [x] having a dedicated branch for the contribution
     - [x] requirement of code coverage
     - [x] validation of checkstyle
     - [x] link to contribution guide
     - [x] command to run locally to validate the contribution

### PR DESCRIPTION
**Pull request by alecharp**
Resolves #202.

### CODE REVIEW

**Comment by alecharp** [comment]
I believe that with https://github.com/jenkins-infra/plugin-health-scoring/pull/203/commits/fa9acb06e42b203536fa5c3fd3affe315b5f4bc9 I'm answering #199.

File: docs/.env.example [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+GITHUB_APP_PRIVATE_KEY_PATH=path-to-private-key
+## The name of the organization or user where the GitHub App was installed
+GITHUB_APP_INSTALLATION_NAME=organization-or-username
+
+# Configuration of the application schedulers
+## Frequency the update-center content is fetched
+## This means on the hour, every 2 hours
+UPDATE_CENTER_CRON=0 0 */2 * * *
+## Frequency the probes and scoring processed are executed
+## This means at 5, every 2 hours
```
Comment (jleon33) [inline docs/.env.example:26]: This is the line that confused me. Does this mean at 5pm, 5am or 5 minutes past the top the hour, every two hours?

Comment (alecharp) [reply docs/.env.example:26]: I see. It's "past 5, every hour", so 2:05am, 4:05am, etc. How would you phrase that?

Comment (jleon33) [reply docs/.env.example:26]: perhaps @kmartens27 could chime in?

Comment (kmartens27) [reply docs/.env.example:26]: I like what @jleon33 shared as his last suggestion. I was thinking of something like
"This means every two hours, five minutes past the top of the hour"

Comment (alecharp) [reply docs/.env.example]: Thank you both. I integrated your suggestion in https://github.com/jenkins-infra/plugin-health-scoring/pull/203/commits/ade5d8a8116c9c4025cfdc7d89b7abd7e5b75906

**Pull request merged**